loris/resolver.py

```diff
@@ -98,34 +106,9 @@ def resolve(self, ident):
         source_fp = self.source_file_path(ident)
         logger.debug('src image: %s' % (source_fp,))
 
-        format = self.format_from_ident(ident)
-        logger.debug('src format %s' % (format,))
-
-        return (source_fp, format)
-
-
-# To use this the resolver stanza of the config will have to have both the
-# src_img_root as required by the SimpleFSResolver and also an
-# [[extension_map]], which will be a hash mapping found extensions to the
-# extensions that loris wants to see, e.g.
-#
-# [resolver]
-# impl = 'loris.resolver.ExtensionNormalizingFSResolver'
-# src_img_root = '/cnfs-ro/iiif/production/medusa-root' # r--
-#   [[extension_map]]
-#   jpeg = 'jpg'
-#   tiff = 'tif'
-# Note that case normalization happens before looking up in the extension_map.
-class ExtensionNormalizingFSResolver(SimpleFSResolver):
```

Comment:
That feels like a breaking (3.x) change to me. I'm headed out for the day, but, briefly, if this class is going away and another class is providing the same functionality, could we leave an alias in place so that the behavior stays the same?

For release notes, I'm really terrible at them, but I try. Could just start a doc somewhere in the repo. That would probably be easier, for sure.